spec/requests/v1/interventions/stars/put_star_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PUT /v1/interventions/:id/star', type: :request do
```

Comment:
When I generate all routes by `rails routes` I got 
`v1_intervention_destroy_star DELETE /v1/interventions/:intervention_id/star(.:format)                                                 v1/interventions/stars#destroy` so I think we should change the name of this file and her description.